andela/ah-backend-athena
# #162163265 send user an email with link to reset password

**What does this PR do?**
send user email with password reset link upon user's request

**Description of the task**
when a user has forgots password they request for password reset. user provides email address and then email with the link is sent.

**How should be manually tested?**
clone the repository and run `pip install -r requirements.txt`
Goto the `/api/password_reset/`
Expected output:
```
{
    "message":"An email has been sent to this email"
}
```
Go to your email and click on the link that has been sent
This will run on this endpoint `/api/password_reset_confirm/<slug>/`
This is what you should receive
```
{
		"password":"qwerty123",
		"confirm_password":"qwerty123"
}
```
Expected output
```
{
    "message": {
        "detail": [
            "Password successfully reset"
        ]
    }
}
```
**NOTE:** Only signed up users can use this service

**What are the relevant pivotal tracker stories?**
#13
